node/service/src/chain_spec/frequency_rococo.rs

```diff
@@ -0,0 +1,101 @@
+#![allow(missing_docs)]
+use cumulus_primitives_core::ParaId;
+use frequency_runtime::{AccountId, AuraId, SudoConfig, EXISTENTIAL_DEPOSIT};
+use sc_service::ChainType;
+
+/// Specialized `ChainSpec` for the normal parachain runtime.
+pub type ChainSpec = sc_service::GenericChainSpec<frequency_runtime::GenesisConfig, Extensions>;
+
+/// The default XCM version to set in genesis config.
+const SAFE_XCM_VERSION: u32 = xcm::prelude::XCM_VERSION;
+
+use super::{get_properties, Extensions};
+
+// pub fn load_frequency_rococo_spec() -> Result<ChainSpec, String> {
+// 	ChainSpec::from_json_bytes(&include_bytes!("../../specs/frequency_rococo.json")[..])
+// }
+
+pub fn frequency_rococo_testnet() -> ChainSpec {
+	let properties = get_properties("UNIT", 12, 42);
+	let para_id: ParaId = 4044.into();
+	ChainSpec::from_genesis(
+		// Name
+		"Frequency",
+		// ID
```

Comment:
want to give any fancy name to show up on rococo testnet? or Frequency is fine

Reply:
I think we will eventually, but for now let's just go with `Frequency Rococo` so that it is clear